<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWidget</class>
 <widget class="QWidget" name="UserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>227</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>User Mode</string>
   </property>
  </widget>
  <widget class="QListWidget" name="DogsList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>261</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Adopt dogs</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Search dogs</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Adoption list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserWidget</receiver>
   <slot>adopt_dogs_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>UserWidget</receiver>
   <slot>search_dogs_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>adopt_dogs_window()</slot>
  <slot>search_dogs_window()</slot>
 </slots>
</ui>
